flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java

```diff
@@ -771,7 +774,7 @@ private static void setManagedMemoryFractionForSlotSharingGroup(
 	private static void setManagedMemoryFractionForOperator(
 			final ResourceSpec operatorResourceSpec,
 			final ResourceSpec groupResourceSpec,
-			final int groupOperatorCount,
+			final long groupOperatorCount,
```

Comment:
Is the int not automatically promoted to a long when calling `BigDecimal.valueOf()`?

Reply:
That's true. I had wanted to keep the type aligned. Will drop this unnecessary change.